src/System.Private.CoreLib/src/Internal/Runtime/Augments/RuntimeThread.Unix.cs

```diff
@@ -20,21 +23,33 @@ public sealed partial class RuntimeThread
 
         internal WaitSubsystem.ThreadWaitInfo WaitInfo => _waitInfo;
 
+#if MONO
+        private void Unix_PlatformSpecificInitialize()
```

Comment:
Yes, the idea is clear for me, but is addition of these files enough for creation of new profiles? For instance, I tried create such files (darwin.dll.sources, win32.dll.source, linux.dll.sources) and they could be built from within mcs/class/corlib folder. But if I try to build them from root mono directory then basic profile building failed. Then I switched on renaming approach (Unix_*, Windows_*) Anyway, I will try to refactor in such way, because this approach is more appropriate.